src/app/components/implementation-view/qpu-selection/qpu-selection.component.ts

```diff
@@ -734,6 +734,22 @@ export class QpuSelectionComponent implements OnInit, AfterViewInit {
       });
   }
 
+  saveResultsToQhana() : void {
+    const request = new XMLHttpRequest();
```

Comment:
the "new" fetch api of browsers is easier to read. (<https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API>)